### PR TITLE
Add recipe for graypy

### PR DESCRIPTION
**Pull request by Anthchirp**

### CODE REVIEW

File: recipes/graypy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - graypy
+
+about:
+  home: https://github.com/severb/graypy
+  license: BSD
```
Comment (CJ-Wright) [inline recipes/graypy/meta.yaml]: Which BSD is it?

Comment (ocefpaf) [reply recipes/graypy/meta.yaml]: Looks like it is BSD-3-Clause, see https://github.com/severb/graypy/blob/master/LICENSE

Comment (Anthchirp) [reply recipes/graypy/meta.yaml]: BSD 3-clause. Will change to ```license: BSD-3-Clause```

https://github.com/severb/graypy/blob/master/LICENSE

**Comment by CJ-Wright** [comment]
LGTM, this is noarch and thus only needs to pass on Circle, thank you very much!

**Pull request merged**